tests/test_apiclient.py

```diff
@@ -21,3 +23,16 @@ def test_constructor_with_success(self):
     def test_constructor_with_no_token(self, token):
         with pytest.raises(ValueError, match='access_token must be provided'):
             RevAiAPIClient(token)
+
+    @pytest.mark.parametrize('error', get_error_test_cases(
+        ['unauthorized', 'job-not-found', 'invalid-job-state']))
+    @pytest.mark.parametrize('method', ["POST", "GET", "DELETE"])
```

Comment:
Do these test cases run as a matrix? ie. each error is run with each method? If so that's pretty convenient :)

Reply:
Yes they do! :)